packages/dog/dog.0.1.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer:   "thomas@gazagnaire.org"
+authors:      "Thomas Gazagnaire"
+license:      "ISC"
+homepage:     "https://github.com/samoht/dog"
+bug-reports:  "https://github.com/samoht/dog/issues"
+dev-repo:     "https://github.com/samoht/dog.git"
+
+build: [
+  ["./configure" "--prefix" prefix "--{alcotest:enable}-tests"]
+  [make]
+]
+build-test: [
+  [make "test"]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "dog"]
+  ["rm" "-f" "%{bin}%/dog"]
+]
```

Comment:
cause I'm too lazy to enumerate all the library files to install :p I'll check if `oasis2opam` can help me.